services/harmony/test/collection-capabilities.ts

```diff
@@ -4,18 +4,24 @@ import { currentApiVersion } from '../app/frontends/capabilities';
 import { hookGetCollectionCapabilities } from './helpers/capabilities';
 import hookServersStartStop from './helpers/servers';
 
+// This is the concept ID of the most recently updated collection with a
+// short name of "harmony_example". There are 3 collections with the same
+// short name, so if either of the other two are updated these tests will no
+// longer pass when regenerating CMR test fixtures.
```

Comment:
Hopefully this is a helpful note. It caught me out for a while where the wrong collection was being returned for this test.